extensions/amp-story/0.1/amp-story.js

```diff
@@ -795,10 +822,31 @@ export class AmpStory extends AMP.BaseElement {
         this.updateBackground_(this.activePage_.element);
       }
     } else {
+      this.vsync_.run({
+        measure: state => {
+          const {offsetWidth, offsetHeight} = this.element;
+          state.isLandscape = offsetWidth > offsetHeight;
+        },
+        mutate: state => {
+          this.toggleLandscapeOverlay_(state.isLandscape);
+        },
+      }, {});
       this.element.removeAttribute('desktop');
     }
   }
 
+  /**
+   * @param {boolean} isLandscape
+   * Toggles landscape overlay based on vsync state.
+   */
+  toggleLandscapeOverlay_(isLandscape) {
```

Comment:
Why not simply use [`ClassList.toggle`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods) instead of a helper method?

Reply:
because breaking it like this helps in testing it

Reply:
Got it. In that case you can just check the DOM state afterwards, no? See comment below.

Reply:
Done